packages/react-native/scripts/cocoapods/__tests__/codegen_utils-test.rb

```diff
@@ -527,7 +535,7 @@ def get_podspec_no_fabric_no_script
           'source' => { :git => '' },
           'header_mappings_dir' => './',
           'platforms' => {
-            'ios' => '11.0',
+            'ios' => '12.4',
```

Comment:
shouldn’t it be also `min_ios_version_supported`?

Reply:
it depends on the philosophy you adopt to write tests.
If you use `min_ios_version_supported` here, once we move the function to the proper file and we reuse the same function of production, this test would be tautological, as changing what the function in prod returns will also change what returned here.

Personally, I prefer to see my test failing if I bump a version in production code, so I can double check that the change was done on purpose and not by mistake. That's why I hardcoded the value.

Does it make sense?

Reply:
interesting – makes sense!